src/index.js

```diff
@@ -3,9 +3,16 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import reportWebVitals from './reportWebVitals';
+import { Normalize } from 'styled-normalize';
+import { GlobalStyle } from './GlobalStyle';
+import { ThemeProvider } from 'styled-components';
+import { theme } from './theme';
 
 ReactDOM.render(
   <React.StrictMode>
+    <ThemeProvider theme={theme}></ThemeProvider>
```

Comment:
Theme provider should wrap around whole application to work properly

Reply:
you're right. I've changed it.